frontend/src/Utils/api-discovery/index.ts

```diff
@@ -32,5 +32,5 @@ export const initAPIDiscovery: InitAPIDiscovery = (storeInstance, preferenceList
 
   // eslint-disable-next-line no-console
   console.info(`API discovery waiting ${API_DISCOVERY_INIT_DELAY} ms before initializing`);
-  window.setTimeout(() => startAPIDiscovery(preferenceList, storeInstance.dispatch), API_DISCOVERY_INIT_DELAY);
+  window.setTimeout(() => startAPIDiscovery(preferenceList, storeInstance.dispatch), resources ? API_DISCOVERY_INIT_DELAY : 0);
```

Comment:
@karelhala I believe this delay was introduced to allow the chrome and application critical resources to load first. Setting to 0 risks us blocking those requests IIRC.

Is there a specific issue you are seeing that this fixes?

Reply:
Chatted with @karelhala offline. Since we moved SDK initialization to scalprum, we are guaranteed that the requests for chrome navigation and chunks will go out first (making the initial API discovery delay less important). 

Right now the experience is pretty rough on first load as it takes 5 seconds to start showing applications. I'm going to approve this as it is an improvement to the end user experience - however we need to revisit this logic all together.